src/components/ui/footer.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button } from "@/components/ui/button";
+import { GradientButton } from "@/components/ui/gradient-button";
+import Image from "next/image";
+import Link from "next/link";
+import Altier from "~/images/logo-sm.png";
```

Comment:
```suggestion
import Image from "next/image";
import Altier from "~/images/logo-sm.png";
```

Supprimer les imports inutiles